lib/audio_service_web.dart

```diff
@@ -0,0 +1,260 @@
+import 'dart:async';
+import 'dart:html' as html;
+import 'dart:ui';
+
+import 'package:audio_service/audio_service.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+
+class Art {
+  String src;
+  String type;
+  String sizes;
+  Art({this.src, this.type, this.sizes});
+}
+
+class AudioServicePlugin {
+  // BackgroundAudioTask _task;
+  int fastForwardInterval;
+  int rewindInterval;
+  Map params;
+  bool started = false;
+  MethodChannel serviceChannel;
+  MethodChannel backgroundChannel;
+
+  static void registerWith(Registrar registrar) {
+    final MethodChannel serviceChannel = MethodChannel(
+      'ryanheise.com/audioService',
+      const StandardMethodCodec(),
+      registrar.messenger,
+    );
+
+    final MethodChannel backgroundChannel = MethodChannel(
+      'ryanheise.com/audioServiceBackground',
+      const StandardMethodCodec(),
+      registrar.messenger,
+    );
+    final AudioServicePlugin instance = AudioServicePlugin();
+    instance.serviceChannel = serviceChannel;
+    instance.backgroundChannel = backgroundChannel;
+    //'ryanheise.com/audioServiceBackground'
+    serviceChannel.setMethodCallHandler(instance.handleServiceMethodCall);
+    backgroundChannel.setMethodCallHandler(instance.handleBackgroundMethodCall);
+  }
+
+  Future<dynamic> handleServiceMethodCall(MethodCall call) async {
+    switch (call.method) {
+      case 'start':
+        fastForwardInterval = call.arguments['fastForwardInterval'];
+        rewindInterval = call.arguments['rewindInterval'];
+        params = call.arguments['params'];
+        started = true;
+        return started;
+      case 'connect':
```

Comment:
One thing this should do is broadcast the current state to the client.

Reply:
Hmmm... So should the plugin be keeping it's own representation of the current state that it can then broadcast? I guess it could fire off the background methods, like the android version does. I will just need to create a `mediaController` on the dart side. 

@ryanheise Since none of this is happening in isolates, the BehaviorSubjects should still be hydrated with current data, so is this actually necessary? (I'm also not clearing things out on disconnect...)

Reply:
I see, that's fair enough. In that case I would at least add a comment explaining this in the connect/disconnect cases.